### ISSUE TITLE
Question on possibilities and understanding

### ISSUE THREAD

**Issue by danield5732** [KIKIRPA/GCMStoolbox#6]
Hello,

the GCMStoolbox looks very usefull to me, however, I want to ask for some details rather examining the code by my own.
For grouping you use MSPEPSEARCH. Is this mandatory? I'm also curious why a peptide search tool is used for your project ELinC dealing with lacquer and resins.
I did not figure out how the grouping is done in particular. For my purpose I'm looking for a tool to compare two or more .MSL files by retention index and the m/z pattern. In short to answer the question if one peak in two measurements is the same substance or not (independent from identification), and which peaks of a reference .MSL are present in a new measurement.
Would this be possible with your tool box, or at least would need small adjustments? 

Te be honest, I did not manage to run the scripts in spyder as they are thought to be run from comand line. This makes it hard to follow the script stepwise. Do you have any helpful hint for that as well?

Sincerely,
Daniel

**Comment by kobbejager** [comment]
Because of the lack of documentation, I was already wondering if you would manage to use these scripts.  We are preparing a scientific paper explaining their use. Meanwhile, I think it would be better to discuss this by e-mail. What is the best way to contact you? I think I have found your tu-berlin address, is that correct?

**Comment by kobbejager** [comment]
To answer on your question on MSPEPSEARCH: while it is developed especially for peptide mass spectral matching, it is essentially just a batch-version of NIST's MS SEARCH application. As such you can use it for mass spectra from all kinds of materials. The advantage over MS SEARCH it that you can search a whole series of mass spectra in one go, collecting all results in a text file.

**Comment by danield5732** [comment]
> 
> 
> Because of the lack of documentation, I was already wondering if you would manage to use these scripts. We are preparing a scientific paper explaining their use. Meanwhile, I think it would be better to discuss this by e-mail. What is the best way to contact you? I think I have found your tu-berlin address, is that correct?

I think you found me :-)
and I would be glad to hear from you!
